bundle/manifests/cc-operator.clusterserviceversion.yaml

```diff
@@ -104,9 +104,9 @@ metadata:
                   "name": "local-bin"
                 }
               ],
-              "payloadImage": "quay.io/confidential-containers/runtime-payload:kata-containers-b11b6e3756a15e602a5bff1d4d47da8ecf23c593",
+              "payloadImage": "quay.io/confidential-containers/runtime-payload-ci:kata-containers-5b7009f2f9cdb39099b97a4f868b53ec9f3c383f",
```

Comment:
Any specific reason to name the runtime payload image as `runtime-payload-ci:kata-containers-ddddd` instead of `runtime-payload:kata-containers-ci-ddddd` ?

Reply:
The reason is that we've been using a different registry for CI specific images, for both Kata Containers and Enclave CC.
Now we've tagged the "official" Kata Containers image, and I've updated it.